package.json

```diff
@@ -286,8 +286,7 @@
     "mv": "^2.1.1",
     "node-abi": "^2.15.0",
     "node-addon-loader": "decred/node-addon-loader#master",
-    "nouislider": "^12.0.0",
-    "pi-ui": "https://github.com/decred/pi-ui",
+    "pi-ui": "https://github.com/guilhermemntt/pi-ui#50b040d1ea0a5e83f6d2fd9d9f3a970282f23320",
```

Comment:
This is a NACK for me.  Needs to be in a decred owned repo.  Or someone with longer standing.

Reply:
Oh, @alexlyp, sorry. Guys at ``pi-ui`` will merge my PR there as soon this one here is approved, but I can talk to them. So I can use the master ``pi-ui`` repo and we can finally merge it if it's ok.

Reply:
OK now that it's merged in pi-ui, please go ahead and remove this change and then should be good to go

Reply:
Done! Thanks @alexlyp